### PR TITLE
Added 3 Window Reacts

### PR DESCRIPTION
**Pull request by caliburn1337**
I added 3 new window reacts, myanimelist, deviantart and netflix. I'm not good at writing the dialogue(mainly because i'm not english), so feel free to change anything.

### CODE REVIEW

File: Monika After Story/game/zz_windowreacts.rpy [hunk 1]
```diff
@@ -688,3 +688,85 @@ label mas_wrs_reddit:
...
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_mal:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "So you like anime and manga, [player]?",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]: There's a player model variable for this, so we should actually be able to know whether or not the player likes this.

File: Monika After Story/game/zz_windowreacts.rpy [hunk 2]
```diff
@@ -688,3 +688,85 @@ label mas_wrs_reddit:
...
+            "Let's watch anime together someday [player]~",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]: Could go for a more gentle tone here
```suggestion
            "Maybe we could watch anime together someday [player]~",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 3]
```diff
@@ -688,3 +688,85 @@ label mas_wrs_reddit:
...
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_deviantart:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "This is known as one of the most famous art sites. So much talent in here!",
+            "This makes me want to learn how to draw!",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            "I'd love to learn how to draw someday...",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 4]
```diff
@@ -688,3 +688,85 @@ label mas_wrs_reddit:
...
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_deviantart:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "This is known as one of the most famous art sites. So much talent in here!",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            "This is known as one of the most famous art sites, there's so much talent in here!",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 5]
```diff
@@ -688,3 +688,85 @@ label mas_wrs_reddit:
...
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_netflix:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "I'd love to watch a romance movie with you [player]!",
+            "Are we going to watch something today?",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            "What are we watching today, [player]?",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 6]
```diff
@@ -688,3 +688,87 @@ label mas_wrs_reddit:
...
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_deviantart:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "This is known as one of the most famous art sites, there's so much talent in here!",
```
Comment (jmwall24) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            "There's so much talent here!",
```
This was a bit wordy for a notification, plus the beginning sounded a bit tour-guidy. Best to keep these as short and sweet as possible.
